README.md

```diff
@@ -1,94 +1,46 @@
 # A Reference Process for OSS Projects on GitHub
 
-This is an attempt to document similarities in in a number of our OSS projects hosted on GitHub. 
+This is an attempt to document practices common to a number of OSS projects we have on GitHub. 
```

Comment:
@pyokagan 

> Maybe this could be part of SE-EDU?

That's another idea with merit. It's not an exact fit, but there is significant overlap between the two. I'm also open for moving it to NUS-OSS or forking it there as @wkurniawan07 suggested. 

Anyway, the revised README claims this repo documents common practices between a number of specific projects. I'll remove the line that says others are welcome to adopt it. After that, it's just a repo in the public domain that happens to have the org name `oss-generic` for some unknown reason. It contains exactly what it claims to contain. I don't think anybody will object to it or make demands for changes. So, for all practical purposes let's treat it as process for our own projects. We can do that without specifically mentioning NUS, the same way we try to keep our other projects (e.g. SE-EDU) independent from NUS. It's important to do so to preserve their freedom to chart their own path in future, instead of being tied to a specific institution. All of these projects are meant to grow beyond NUS, although they are currently concentrated in NUS. The process documented in OSS-Generic is not mature yet, so I don't think anybody would want to adopt it anyway. We can revisit this topic perhaps one year down the line and see if we have something we can offer to the rest of the world.